LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -12,14 +12,16 @@ public class SpinnerController : Controller
         //DIJ in 4 STEPS -
         //TODO: 0) Register the Repository class as a service in Program.cs 
         //TODO: 1) add an instance variable here of type Repository
-
+        private Repository repository;
 
         /***
          * Constructor - TODO: 2) call for a DIJ Repository object to be passed to the constructor
          **/
-        public SpinnerController()
+
+        public SpinnerController(Repository bob)
```

Comment:
This works fine; you probably will want to call this something more self-explanatory, rather than `bob`